modules/gis/docs/README.md

```diff
@@ -0,0 +1,15 @@
+# @loaders.gl/gis
+
+This module contains helper classes for the GIS category of loaders.
+
+## Installation
+
+```bash
+npm install @loaders.gl/gis
+```
+
+## Loaders and Writers
+
+| Loader                                                    |
+| --------------------------------------------------------- |
+| [`MVTLoader`](modules/mvts/docs/api-reference/mvt-loader) |
```

Comment:
Look at the table in the top docs/README for a list of gis loaders (or even the main doc page of https://loaders.gl)

Reply:
For now I removed this section, because `modules/gis` isn't itself a loader.